docs/devGuide/development/migratingNodeJs.md

```diff
@@ -0,0 +1,95 @@
+{% set title = "Migrating Node.js" %}
+<span id="title" class="d-none">{{ title }}</span>
+
+<frontmatter>
+  title: "{{ title }}"
+  layout: devGuide.md
+  pageNav: default
+</frontmatter>
+
+# {{ title }}
+
+<div class="lead">
+
+Node.js versions have to upgraded periodically before the version being used reaches its [end of life](https://endoflife.date/nodejs).
+This page outlines the steps of migrating to a higher version of Node.js
+</div>
+
+### Preparation
+
+#### Choosing Node.js version
+Before upgrading the Node.js version, it is important to decide which version to upgrade to. 
+Broadly speaking, migrate only to even-numbered Node.js releases as only even-numbered versions will be provided with Long-Term Support. 
+
+Read more about Node.js release lines [here](https://nodesource.com/blog/understanding-how-node-js-release-lines-work/).
+
+npm version will be upgraded accordingly to the Node.js version. Hence, do check that this upgrade does not cause any issues.
+<box type="info" seamless>
+If a different version of npm is needed this can be overridden. This will be explained further below.
+</box>
+
+#### Setup 
+
+Install <tooltip content="Node Version Manager">[nvm](https://github.com/nvm-sh/nvm)</tooltip> to help switch between different node versions.
+Switch to use the Node.js version that you are migrating to.
+
+### Migration steps 
+
+1. Refactor any deprecated syntax
+   - Go to the [Node.js changelog](https://nodejs.org/en/blog/release) of the new version
+   - Go through list of deprecated syntax and check if it is being used in Markbind
```

Comment:
```suggestion
   - Go through the list of deprecated syntax and check if it is being used in MarkBind
```

Let's standardise to use "MarkBind" (instead of "Markbind") throughout! Same for the other occurrences in this file :)